collections/conf/routes

```diff
@@ -10,6 +10,7 @@ GET     /collections                                    controllers.CollectionsC
 POST    /collections                                    controllers.CollectionsController.addChildToRoot
 POST    /collections/*collection                        controllers.CollectionsController.addChildToCollection(collection: String)
 DELETE  /collections/*collection                        controllers.CollectionsController.removeCollection(collection: String)
+GET     /corrected-collections                          controllers.CollectionsController.correctedCollections
```

Comment:
I know it's just a temporary route, but `POST` would be better (`GET` isn't supposed to have side-effects)